packages/admin-ui/ui/src/domain/product-categories/components/product-categories-list.tsx

```diff
@@ -53,15 +54,18 @@ function ProductCategoriesList(props: ProductCategoriesListProps) {
       }
 
       try {
+        isError && setIsError(false)
```

Comment:
```suggestion
        setIsError(false)
```
won't rerender if already `false`